ft_math.m

```diff
@@ -480,6 +477,17 @@
 ft_postamble history    data
 ft_postamble savevar    data
 
+
+function assign_var(var, val)
+% Note: using an anonymous function as follows does not work in Octave:
```

Comment:
can you document this in the code guidelines at http://www.fieldtriptoolbox.org/development/guidelines/code